zenodo/modules/spam/views.py

```diff
@@ -105,6 +105,87 @@ def delete(user_id):
         ctx.update(records=records)
         return render_template('zenodo_spam/delete.html', **ctx)
 
+def user_info(owners):
+    users = (
+        User.query
+        .options(db.joinedload(User.profile), db.joinedload(
+            User.external_identifiers))
+        .filter(User.id.in_(owners))
+    )
+
+    data = {}
+
+    for user in users:
+        info = {
+            "id": user.id,
+            "email": user.email,
+            "full_name": user.profile.full_name,
+            "username": user.profile.username,
+            "last_records": "",
+            "external_identifiers": "",
+        }
+
+        # TODO maybe make it more beautiful?
+        try:
+            if user.external_identifiers:
+                for identifier in user.external_identifiers:
+                    info["external_identifiers"] = (
+                        info["external_identifiers"] + identifier.method + ", "
+                    )
+                info["external_identifiers"] = info["external_identifiers"][
+                    :-2
+                ]
```

Comment:
I would try to pass more structured data and then delegate the display in the template.

```suggestion
            if user.external_identifiers:
                info["external_identifiers"] = [i.method for i in user.external_identifiers]
```